codegenerator/cli/templates/static/codegen/src/db/TablesStatic.res

```diff
@@ -134,9 +134,9 @@ module RawEvents = {
       mkField("src_address", Text),
       mkField("block_hash", Text),
       mkField("block_timestamp", Integer),
-      mkField("block_fields", Json),
-      mkField("transaction_fields", Json),
-      mkField("params", Json),
+      mkField("block_fields", JsonB),
+      mkField("transaction_fields", JsonB),
+      mkField("params", JsonB),
```

Comment:
Isn't `Json` cheaper then `Jsonb`?

Reply:
It makes little difference to performance but Jsonb is slightly faster for entity history. I'll do more in depth testing.